hack/update-api-reference-docs.sh

```diff
@@ -56,13 +57,25 @@ if [[ $(uname) == "Darwin" ]]; then
   user_flags=""
 fi 
 
-docker run ${user_flags} --rm -v $V1_TMP_IN_HOST:/output:z -v ${SWAGGER_PATH}:/swagger-source:z gcr.io/google_containers/gen-swagger-docs:v4.2 \
-    v1 \
-    https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/api/v1/register.go
-
-docker run ${user_flags} --rm -v $V1BETA1_TMP_IN_HOST:/output:z -v ${SWAGGER_PATH}:/swagger-source:z gcr.io/google_containers/gen-swagger-docs:v4.2 \
-    v1beta1 \
-    https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/apis/extensions/v1beta1/register.go
+for ver in $VERSIONS; do
+  TMP_IN_HOST="${OUTPUT_TMP_IN_HOST}/${ver}"
+  REGISTER_FILE_URL="https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg"
+  if [[ ${ver} == "v1" ]]; then
+    REGISTER_FILE_URL="${REGISTER_FILE_URL}/api/${ver}/register.go"
+  else
+    REGISTER_FILE_URL="${REGISTER_FILE_URL}/apis/${ver}/register.go"
+  fi
+  # TODO: we need to rename the swagger JSON files, currently they are named as
```

Comment:
Yes, this is necessary, since batch is starting out in v1. Is it possible to do it in this PR?